google/resumable_media/requests/download.py

```diff
@@ -156,6 +157,8 @@ def consume(self, transport):
                 finished.
         """
         method, url, payload, headers = self._prepare_request()
+        if self.user_agent is not None:
+            headers[u'User-Agent'] = self.user_agent
```

Comment:
Not sure which way is better, to have the `if` check here or simply allow `headers[u'User-Agent']` have a `None` value.

Reply:
We should not set user_agent as None if api caller doesnt provide user-agent. In that case, user_agent would be 'python-requests/2.18.' which is set by the underlying requests lib.